fetch/api/response/response-error-from-stream.html

```diff
@@ -8,33 +8,31 @@
   </head>
   <body>
     <script>
-      function CustomTestError() {
-        const error = Error();
-        error.name = 'custom-test-error';
-        return error;
-      }
-
       function newStreamWithStartError() {
-        return new ReadableStream({
-          start(controller) {
-            controller.error(CustomTestError());
-          }
-        })
+        var err = new Error("Start error " + Math.random());
```

Comment:
@bzbarsky this causes flakiness in the Chromium CQ, since the error message is not deterministic (so multiple runs that FAIL will have different error messages). 

What was the purpose of the `Math.random()` here?

Reply:
(Similarly with the Pull error below)

Reply:
It was to differentiate the different Error instances, but we could just remove that bit: the test is using promise_rejects_exactly anyway after these changes.  I will do that.

Reply:
https://github.com/web-platform-tests/wpt/pull/21641